### DISCUSSION TITLE
Masking fisheye photos for HDRI

### DISCUSSION THREAD

**Discussion by jmws219** [alicevision/Meshroom#2540] [General]
Hi Everyone,
I am using Meshroom for the first time and trying to create a 360 Hdri from some fisheye pictures. Meshroom seems to have worked out a good line up, but I had a lens hood attached, which I need to mask it out as it is visible on each side, but I can't work out how to do it. I have created a black and white png in photoshop, and I think I should be using the Masks folder in the feature extraction node, but it would be great to have some help to get it working.
Thanks

**Comment by natowi** [comment]
Take a look at this mini masking tutorial: https://github.com/alicevision/Meshroom/wiki/New-Features-in-Meshroom-2023.1#1-image-masking

You should be able to use the "Mask Folder" as input instead of the masking node. Take note of the naming convention.

**Reply by natowi** [reply]
And conveniently a tutorial was just released that includes masking: https://www.youtube.com/watch?v=XUKu1apUuVE

**Reply by jmws219** [reply]
Amazing. Thank you so much for linking these.
